public/assets/js/scripts.js

```diff
@@ -294,6 +294,17 @@ $(document).ready(function () {
     $(this).html() == '<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>' ? $(this).html('<span><img src="./assets/img/icons/view-map-pin.svg" alt="view map"> &nbsp; View Map</span>') : $(this).html('<span><img src="./assets/img/icons/close-map-pin.svg" alt="view map"> &nbsp; Close Map</span>');
   });
 
+
+  // disable calculate button 
+  $('.periodic-investment').change(function() { 
```

Comment:
Can you add an events for the input text fields too? I noticed that the button is still deactivated when the price is changed from the text field only (without using the range slider)